packages/components/Callout/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.json",
```

Comment:
Can we use a package name syntax rather than relative path?

Reply:
I'll check this out; I was going off of another package's tsconfig for reference